ui/src/app/views/workspace-edit/component.ts

```diff
@@ -89,8 +91,19 @@ export class WorkspaceEditComponent implements OnInit {
     this.locationService.back();
   }
 
-  resetWorkspaceCreation(): void {
+  reloadConflictingWorkspace(): void {
+    this.workspacesService.getWorkspace(this.oldWorkspaceNamespace,
+                                        this.oldWorkspaceName)
+      .subscribe((workspace) => {
+        this.workspace = workspace;
+      });
```

Comment:
Can we abstract the fetching out into its own function now that its being used in multiple places? (init and reload?)